src/server/views/layout.html

```diff
@@ -36,18 +36,19 @@
 
 {% block beforeContent %}
     {% if phaseTag %}
-        {% if feedbackLink %}
+        {% if 'mailto:' in feedbackLink %}
             {{ govukPhaseBanner({
                 tag: {
                     text: phaseTag | capitalize
                 },
-                html: 'This is a new service – your <a class="govuk-link" href=' + feedbackLink + '>feedback</a> will help us to improve it.'
+                html: 'This is a new service. Help us improve it and <a class="govuk-link" href="' + feedbackLink + '">give your feedback by email</a>.'
             }) }}
         {% else %}
             {{ govukPhaseBanner({
                 tag: {
                     text: phaseTag | capitalize
-                }
+                },
+                html: 'This is a new service. Help us improve it and <a class="govuk-link" href="' + feedbackLink + '">give your feedback (opens in new tab)</a>.'
```

Comment:
Probably the same here

We set up a nice [Nunjucks helper](https://github.com/DEFRA/forms-designer/blob/3d884bea6ad33cd42eb71dd83a9b19317ab3cb7e/designer/server/src/models/forms/form-lifecycle.js#L30-L33) for this in **forms-designer** which could be handy (another day)

Reply:
Fixed in the following two commits.